python/hyperon/runner.py

```diff
@@ -0,0 +1,65 @@
+import os
+import hyperonpy as hp
+from .atoms import Atom
+from .base import GroundingSpace, Tokenizer, SExprParser
+from . import stdlib
+
+class MeTTa:
+
+    def __init__(self, space = None, cwd = "."):
+        if space is None:
+            space = GroundingSpace()
+        self.cmetta = hp.metta_new(space.cspace, cwd)
+        stdlib.import_to(self)
+
+    def __del__(self):
+        hp.metta_free(self.cmetta)
+
+    def space(self):
+        return GroundingSpace._from_cspace(hp.metta_space(self.cmetta))
+
+    def tokenizer(self):
+        return Tokenizer._from_ctokenizer(hp.metta_tokenizer(self.cmetta))
+
+    def add_token(self, regexp, constr):
```

Comment:
It probably makes sense to remove `add_token` from `MeTTa` and use `metta.tokenizer().register_token(...)` instead. The similar to `add_atom` we could move it into `Tokenizer` and use via `metta.tokenizer()`.